rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -63,7 +64,8 @@
                         <thorntail.http.port>${thorntail.http.port}</thorntail.http.port>
                     </environment>
                     <properties>
-                        <lra.http.port>${thorntail.http.port}</lra.http.port>
+                        <lra.http.port>${lra.http.port}</lra.http.port>
+                        <thorntail.http.port>${thorntail.http.port}</thorntail.http.port>
```

Comment:
I am not sure whether it is really necessary to override properties with the same name. Can you explain why this is needed?

Reply:
lra.http.port is the name of a Java system property. The value passed in to the plugin configuration (${lra.http.port}) is a maven property that is used to initialise the Java system property. In addition the property setting is part of the thorntail-maven-plugin configuration and makes it clear how and where properties are being set. Furthermore, the more clarity that can be injected into software the fewer will be the resulting bugs - it is an essential part of the art of programming.